### DISCUSSION TITLE
Is there a way I can claim an username squatted by an unused account ?

### DISCUSSION THREAD

**Discussion by bemanisoundteam** [community/community#27567] [Other Feature Feedback, Questions, & Ideas]
Hello,

There is a username that I would like to use, but it's claimed by an account whose only (visible but I doubt there's hidden) activity is the account creation.

I read on the username policy that GitHub prohibits name squatting and usernames may not be inactivity held.

Thanks in advance,
$username

**Comment by radar-blip** [comment]
I remember reading somewhere, its not possilble. If I can find where I read that, I'll post it back here.
I didn't read it all but you can have a look, here: #23495

**Comment by bemanisoundteam** [comment]
I settled with a pun, I like it less but it's the best option I currently have

**Reply by radar-blip** [reply]
What pun is that?

**Reply by bemanisoundteam** [reply]
There's an anime character called Megumi Kato, and in Japanese Megumi means blessing, but since Meg**a**mi means goddess, I tried to use this letter swap, it's not that much of a pun I know

**Reply by radar-blip** [reply]
I remember Kato from The Green Hornet, played by Bruce Lee 😃

**Comment by jfullstackdev** [comment]
you will see a lot of similar posts here and the Staff members already answered

<https://github.com/orgs/community/discussions?discussions_q=name+squatting>
